packages/editor/src/plugins/fill-in-the-blanks-exercise/renderer.tsx

```diff
@@ -1,4 +1,4 @@
-import { DndWrapper } from '@editor/core/components/dnd-wrapper'
+import dynamic from 'next/dynamic'
```

Comment:
using nextjs features in the editor breaks the editor for external integrations

Reply:
Thanks, that was my bad...